### PR TITLE
Adding Battle! for Loot game

### PR DESCRIPTION
**Pull request by josephbedminster**
Adding Battle! for Loot game.

Battle! for Loot is a brand new PvP mechanism for the Loot Metaverse. Fight against multiple ennemies to win Loot Craft Bags 💰💎

https://loot-craft.com/battle

Best,
Joseph

## Project Name

Etherscan link: https://etherscan.io/address/ 

Website: https:// 

Twitter: https://twitter.com/ 

Discord: https://discord.com/

### CODE REVIEW

File: utils/lists.ts [hunk 1]
```diff
@@ -227,6 +227,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Adventurers with distinct but flexible aspects for inspiring stories",
     url: "https://opensea.io/collection/adventurersproject",
   },
+  {
+    name: "⚔️ Battle! (for Loot)",
```
Comment (zhew2013) [inline utils/lists.ts:231]: Move it below project Banquet to follow alphabetic ordering?

Comment (josephbedminster) [reply utils/lists.ts]: Indeed 👀
Thanks

File: utils/lists.ts [hunk 2]
```diff
@@ -227,16 +227,16 @@ export const derivativesList: Record<string, string>[] = [
...
-    url: "https://loot-craft.com/battle",
-  },
   {
     name: "Banquets",
     description: "Randomized adventurer meals",
     url: "https://etherscan.io/address/0x615a610649e656485d9baf0ebe525496d7b78e24",
   },
+  {
+    name: "⚔️ Battle! (for Loot)",
+    description: "Battle! for Loot is a brand new PvP mechanism for the Loot Metaverse. Fight against multiple ennemies to win Loot Craft Bags 💰💎",
```
Comment (ravindranrahul) [inline utils/lists.ts:237]: The description is too long. Can you trim it down so it fits in one line like others?

**Review by raxhvl — CHANGES_REQUESTED** [review]

File: utils/lists.ts (hunk 2 above)
Comment (josephbedminster) [reply utils/lists.ts]: Done!

**Comment by ravindranrahul** [comment]
Thanks! the site looks cool btw

**Pull request merged**